proto/osmosis/claim/v1beta1/genesis.proto

```diff
@@ -51,7 +52,7 @@ message GenesisState {
   ];
   // the maximum claimable amount per address
   // [(account_string, max_claimable), (account_string, max_claimable) ... ]
-  repeated cosmos.bank.v1beta1.Balance claimables = 5
+  repeated cosmos.bank.v1beta1.Balance initial_claimables = 5
```

Comment:
Can the comments be updated here to better explain the relation between initial claimable and that its the maximum?